release: v0.17.3 (#1695)
apollographql/apollo-cli: CHANGELOG.md

## 📚 Documentation -->

# [0.17.3] - 2023-08-01

## 🚀 Features

- **Support binary mirrors via the NPM installer - @LongLiveCHIEF, #1675**

>   When installing Rover via NPM, you can override the base URL where Rover binaries are downloaded from by setting the `APOLLO_ROVER_DOWNLOAD_HOST` environment variable or adding the following to your global or local `.npmrc`:

Reviewer: Just trying to break up the sentence a little for easier comprehension:

```suggestion
  When installing Rover via NPM, you can override the base URL where Rover binaries are downloaded from in one of two ways:
 - by setting the `APOLLO_ROVER_DOWNLOAD_HOST` environment variable
 - by adding the following to your global or local `.npmrc`:
```

>   When installing Rover via NPM, you can override the base URL where Rover binaries are downloaded from by setting the `APOLLO_ROVER_DOWNLOAD_HOST` environment variable or adding the following to your global or local `.npmrc`:

Reviewer: ```suggestion
  When installing Rover via npm, you can override the base URL where Rover binaries are downloaded from by setting the `APOLLO_ROVER_DOWNLOAD_HOST` environment variable or adding the following to your global or local `.npmrc`:
```

npm doesn't capitalize npm, so will follow their lead :)

  
  ```ini
  apollo_rover_download_host=https://your.mirror.com/repository
  ```

>   This enables installing Rover when `https://rover.apollo.dev` is unavailable (e.g., in a private network).

Reviewer: ```suggestion
  This enables installing Rover when `https://rover.apollo.dev` is unavailable, for example, in a private network.
```

(I generally [prefer English terms over Latin abbreviations](https://developers.google.com/style/abbreviations#:~:text=Prefer%20English%20terms%20over%20Latin,in%20some%20circumstances.).)
